ilc/client/ClientRouter.js

```diff
@@ -100,10 +112,16 @@ export default class ClientRouter extends EventEmitter {
 
         //Todo: side effect have to be removed from getter
         let willBe = slotWillBe.default;
```

Comment:
Maybe it make sense to rename slotWillBe vairable to transitionType\transitionAction ?

Reply:
I do not want to overcomplicate current PR because it is not tiny anyhow